_data/icons/edexa.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmeZW6VKUFTbz7PPW8PmDR3ZHa6osYPLBFPnW8T5LSU49csdf",
+    "width": 400,
+    "height": 400,
+    "format": "png"
```

Comment:
✗ ~/tmp ipfs get QmeZW6VKUFTbz7PPW8PmDR3ZHa6osYPLBFPnW8T5LSU49csdf      19:35:51
Error: invalid path "QmeZW6VKUFTbz7PPW8PmDR3ZHa6osYPLBFPnW8T5LSU49csdf": selected encoding not supported

Reply:
Hi @ligi  i have runneded with pretier still shows changes required

Reply:
commited back with running prettier again

Reply:
all checks have been passed it can be merged now right.?! @ligi

Reply:
need to just add the image in the iconsdownload in cid name without extension right

Reply:
yes like the others

Reply:
running prettier adding new line to the files at the end i think i have fixed it

Reply:
hi please review and accept changes @ligi

Reply:
![image](https://user-images.githubusercontent.com/111600/209134082-8da091ce-6b8a-4cc8-a27c-b7a77302ff72.png)

will not review with so many files changed ..

Reply:
reverted back the commits and runned the prettier